dmoj/settings.py

```diff
@@ -32,8 +33,86 @@
 SITE_ID = 1
 SITE_NAME = 'DMOJ'
 SITE_LONG_NAME = 'DMOJ: Modern Online Judge'
+SITE_ADMIN_EMAIL = False
+
+# Constants
+DMOJ_SSL = 0
```

Comment:
Add comment for what the possible values here are (https://github.com/DMOJ/site/pull/1079/files#diff-43f2be90fc113eb0e09a185fcfc97b69L20)